bugwarrior/services/gitlab.py

```diff
@@ -667,6 +670,16 @@ def get_all_repos(self):
         repos = list(filter(self.filter_repos, all_repos))
         return repos
 
+    def description(self, issue):
+        description = issue['description']
+
+        if description:
+            description = description.replace('\r\n', '\n')
```

Comment:
I think this line should be dropped because it is unrelated to this feature. I don't recall the history or purpose behind it, but you can see in the diff of #782 that the github service was already doing this replacement before introducing the body_length option.